### ISSUE TITLE
[$15][MSFT-246] [Registration] High contrast : IE11 : Link images are NOT loading

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2899]
**Steps to Reproduce**
1. Set Windows machine in High contrast mode
2. Launch IE11
3. Launch https://accounts.topcoder.com/member/registration

**Expected Result**
All images should be visible in high contrast mode

**Actual Result**
Note that link image in "registration with" area are not visible in high contrast mode 

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: IE 11

**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.3 Contrast (minimum)
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=1a9ec9e4%2D985a%2D449c%2D9e7a%2Dbb83cbe69ccb&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FArun%5FA11Y%2FHigh%20contrast%20IE11

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096232 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096232 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096232 has been updated - it has been assigned to nac]-[i.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096731 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30096731 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
@veshu this ticket prize is also yet to be doubled.

**Comment by r0hit-gupta** [comment]
Already fixed by PR https://github.com/appirio-tech/accounts-app/pull/251

**Comment by crazyk07** [comment]
QA Reviewer: This should have already been fixed by a merge in production, please check

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30096731<br/><br/>```This is an automated message for crazyk via Topcoder X```
